packages/quick_actions/quick_actions/pubspec.yaml

```diff
@@ -17,15 +17,18 @@ dependencies:
   flutter:
     sdk: flutter
   meta: ^1.3.0
+  quick_actions_platform_interface: ^1.0.0
 
 dev_dependencies:
-  test: ^1.16.3
   flutter_test:
     sdk: flutter
   integration_test:
     sdk: flutter
-  pedantic: ^1.10.0
+  mockito: ^5.0.0-nullsafety.7
+  pedantic: ^1.11.0
+  plugin_platform_interface: ^2.0.0
+
 
 environment:
-  sdk: ">=2.12.0-259.9.beta <3.0.0"
-  flutter: ">=1.12.13+hotfix.5"
+  sdk: ">=2.12.0 <3.0.0"
+  flutter: ">=2.0.0"
```

Comment:
Any reason why you bumped the `flutter` SDK version to `>=2.0.0`. I think this should still work fine on the `1.12.13+hotfix.5` version and keeps it inline with other plugins.